Use RetainPtr in NetworkDataTask constructor

achristensen07 opened the pull request:
#### 74625369100c3fdb04dc7c00395fa5d2c4684abc
<pre>
Use RetainPtr in NetworkDataTask constructor
<a href="https://bugs.webkit.org/show_bug.cgi?id=242821">https://bugs.webkit.org/show_bug.cgi?id=242821</a>
rdar://91340995

Reviewed by David Kilzer, Chris Dumez and Geoffrey Garen.

This shouldn&apos;t be necessary, but it certainly can&apos;t make things worse.

* Source/WebKit/NetworkProcess/cocoa/NetworkDataTaskCocoa.mm:
(WebKit::NetworkDataTaskCocoa::NetworkDataTaskCocoa):

Canonical link: <a href="https://commits.webkit.org/252528@main">https://commits.webkit.org/252528@main</a>
</pre>

cdumez reviewed (approved):
Ok.

geoffreygaren reviewed (approved):
r=me

ddkilzer reviewed (approved):
It's What Would ARC Do (if it were enabled in WebKit).

Commit:
Use RetainPtr in NetworkDataTask constructor
https://bugs.webkit.org/show_bug.cgi?id=242821
rdar://91340995

Reviewed by David Kilzer, Chris Dumez and Geoffrey Garen.

This shouldn't be necessary, but it certainly can't make things worse.

* Source/WebKit/NetworkProcess/cocoa/NetworkDataTaskCocoa.mm:
(WebKit::NetworkDataTaskCocoa::NetworkDataTaskCocoa):

Canonical link: https://commits.webkit.org/252528@main
